src/main/java/algorithms/curated170/medium/threesumsmaller/ThreeSumSmallerRecursion.java

```diff
@@ -0,0 +1,48 @@
+package algorithms.curated170.medium.threesumsmaller;
+
+import java.util.Arrays;
+
+public class ThreeSumSmallerRecursion {
+
+    int[] nums;
+    int target;
+    int rightBoundary;
+
+    public int threeSumSmaller(int[] nums, int target) {
+
```

Comment:
It is also great, after fixing the if check:
![image](https://user-images.githubusercontent.com/60903744/121156794-b20c7880-c851-11eb-9e74-4f99344b2b5c.png)